scripts/sanity_check.sh

```diff
@@ -1,19 +1,24 @@
 #!/bin/bash
 
 UNAME=$(uname -m)
+ROUTER_MANAGED='yes'
 case "$UNAME" in
   "x86_64")
     PLATFORM='gold'
     ;;
   "aarch64")
     if [[ -e /etc/firewalla-release ]]; then
       PLATFORM=$( . /etc/firewalla-release 2>/dev/null && echo $BOARD || cat /etc/firewalla-release )
+      if [[ $PLATFORM == "blue" ]]; then
+        ROUTER_MANAGED='no'
+      fi
     else
       PLATFORM='unknown'
```

Comment:
for "unknown" platform, maybe need set ROUTER_MANAGED='no' as well ?

Reply:
probably all future platforms will have FireRouter, we write less code with this change

Reply:
In normal cases, all future platforms would have `/etc/firewalla-release` in place,  which would have the default value of 'yes' for ROUTER_MANAGED as expected.

However, if something unexpected happens to cause file `/etc/firewalla-release` lost, the "else" block would have PLATFORM set to "unknown". Then do we still want to leave **ROUTER_MANAGED** as "yes" in that case?

But it is an exceptional case, where many things would have been broken anyway. So feel free to leave it as is

Reply:
yes, worst case is broken sanity check anyway. This is just trying to avoid writing code for future platforms.